tokens/staking/stake.md

```diff
@@ -1,6 +1,6 @@
 ---
-title: 如何进行质押
-description: 本教程将展示如何通过委托收集人在Moonbeam质押Token
+title: 如何质押您的MOVR和GLMR代币
+description: 了解如何通过委托收集人在Moonbeam上质押代币并获得奖励。
 ---
 
```

Comment:
line 293 should use this updated URL /learn/features/staking/#reward-distribution